vpr/src/timing/read_sdc.cpp

```diff
@@ -741,13 +742,13 @@ class SdcParseCallback : public sdcparse::Callback {
         VTR_ASSERT_MSG(capture_clock.period >= 0., "Clock period must be positive");
 
         float constraint = std::numeric_limits<float>::quiet_NaN();
-
-        if (std::fabs(launch_clock.period - capture_clock.period) < EPSILON && std::fabs(launch_clock.rise_edge - capture_clock.rise_edge) < EPSILON && std::fabs(launch_clock.fall_edge - capture_clock.fall_edge) < EPSILON) {
+        constexpr float float_epsilon = std::numeric_limits<float>::epsilon();
+        if (std::fabs(launch_clock.period - capture_clock.period) < float_epsilon && std::fabs(launch_clock.rise_edge - capture_clock.rise_edge) < float_epsilon && std::fabs(launch_clock.fall_edge - capture_clock.fall_edge) < float_epsilon) {
```

Comment:
We could change these to use vtr::isclose () https://docs.verilogtorouting.org/en/latest/api/vtrutil/other/#_CPPv4I0EN3vtr7iscloseEb1T1T1T1T